docker-entrypoint.sh

```diff
@@ -2,6 +2,13 @@
 set -e
 set -o allexport
 
+# Backup current env
+env > /tmp/environment.env
```

Comment:
@frankh what's the reason for the env dance? I guess this means there will be things that we will not be able to override by adding to default.env , is that desired?